C2CRIBuildDir/projects/C2C-RI/src/jameleon-test-suite-3_3-RC1-C2CRI/jameleon-core/src/java/net/sf/jameleon/ui/TestCasePane.java

```diff
@@ -291,8 +296,18 @@ public void run() {
                         /**
                          * Change the output of logging messages to go to the new log file.
                          */
+						DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss");
+                        Date date = new Date();
+						String fileName = fn.getFile().toString() + "-" + dateFormat.format(date);
```

Comment:
What would the new filename be?

Reply:
It appends the date to the file name of test case